packages/legend-query-builder/src/components/QueryBuilderSideBar.tsx

```diff
@@ -156,257 +153,57 @@ export const QueryBuilderClassSelector = observer(
 
     return (
       <div className="query-builder__setup__config-group query-builder__setup__config-group--class">
-        <div className="query-builder__setup__config-group__content">
-          <div className="query-builder__setup__config-group__item ">
-            <div
-              className="btn--sm query-builder__setup__config-group__item__label"
-              title="class"
-            >
-              <PURE_ClassIcon />
-            </div>
-            <CustomSelectorInput
-              className="panel__content__form__section__dropdown query-builder__setup__config-group__item__selector query-builder__setup__config-group__item__selector__milestoned"
-              placeholder={
-                classOptions.length
-                  ? 'Choose a class...'
-                  : noMatchMessage ?? 'No class found'
-              }
-              disabled={
-                classOptions.length < 1 ||
-                (classOptions.length === 1 && Boolean(selectedClassOption))
-              }
-              noMatchMessage={noMatchMessage}
-              options={classOptions}
-              onChange={changeClass}
-              value={selectedClassOption}
-              darkMode={
+        <div className="query-builder__setup__config-group__item ">
+          <label
+            className="btn--sm query-builder__setup__config-group__item__label"
+            title="entity"
+            htmlFor="query-builder__setup__class-selector"
+          >
+            Entity
+          </label>
+          <CustomSelectorInput
+            inputId="query-builder__setup__class-selector"
+            className="panel__content__form__section__dropdown query-builder__setup__config-group__item__selector query-builder__setup__config-group__item__selector__milestoned"
+            placeholder={
+              classOptions.length
+                ? 'Choose a class...'
```

Comment:
if we rename it to Entity, shall we change this too?

Reply:
Done